src/RK45.jl

```diff
@@ -371,14 +372,18 @@ function stepcontrolPI!(s)
     β2 = -1/5 / 5
     ε = 0.8
     if s.ok
-        s.errlast == 0 && (s.errlast = s.err)
-        fac = s.safety * (ε/s.err)^β1 * (ε/s.errlast)^β2
+        s.errlast == 0 && (s.errlast = s.err) # if last error is zero, use current error instead
+        if s.errlast == 0 || s.err == 0
```

Comment:
why do you need the `||` here? if `s.errlast == 0` then it gets reset in the line above. So we just need `if s.err == 0`?

Reply:
I was too lazy to think throught the possible cases.